Src/Plugins/AzureDevOpsConfiguration/AzureDevOpsConfiguration.csproj

```diff
@@ -11,6 +11,12 @@
     <PackageId>Sarif.PatternMatcher.AzureDevOpsConfiguration</PackageId>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <MSBuildWarningsAsMessages>
```

Comment:
@schlaman-ms Can you show the before and after due to this change? The build will now result in these messages? Or will they be suppressed entirely?

Reply:
No messages will show in the build at all with these suppressions.
Before you had many many lines of warning from the Newtonsoft conflict. Then there were the 4 copy warning lines.
Now there are zero lines generated as a result of these suppressions.

Reply:
Not sure if this actually resolves the underlying problem.

Reply:
Michael wanted me to suppress the Newtonsoft conflict warnings.

As for the 4 files copying issue (that looks like it has been there for a couple of years). All this does is suppress the message that on at least one of the copies it fails (which is why the continue on error=true is there and only on those 4 copies).
This just suppresses the warning but doesn't affect the proper copying of the files.

Reply:
When you disable warnings in project files, how about adding an XML comment describing what's being suppressed?

Reply:
Added very complete XML comments to explain the 2 suppressions.

Reply:
So, this is an appeal to authority, and I encourage everyone to reject comments on the basis of 'Michael said so'. Instead, please try to communicate the technical reason I provided: the SARIF SDK is consumed by other projects that themselves may consume outdated versions of Newtonsoft. If we update our references, we will force those projects to upgrade as well, which may invalidate our assembly from reuse. 

I say lots of things, and I'm often wrong or my information is incomplete. Trust me if you haven't had direct experience of it already.

Reply:
So a better answer would be:

We have consumers of this tool that use older frameworks and if we updated to newer versions of Newtonsoft then that would break the compatibility with consumers that are not as up to date.
Because of that, it is necessary to maintain older versions within the tool and this creates warnings that there are version conflicts. For this reason I suppressed the warnings rather than update the versions.

Reply:
I think this is wrong, the correct xml comment should be: `<!-- ... -->` ?